spec/HttpClientPool/LeastUsedClientPoolSpec.php

```diff
@@ -69,25 +69,4 @@ public function it_reenable_client(HttpClient $client, RequestInterface $request
         $this->shouldThrow('Http\Client\Exception\HttpException')->duringSendRequest($request);
         $this->shouldThrow('Http\Client\Exception\HttpException')->duringSendRequest($request);
     }
-
-    public function it_uses_the_lowest_request_client(HttpClientPoolItem $client1, HttpClientPoolItem $client2, RequestInterface $request, ResponseInterface $response)
```

Comment:
Are we okey to remove this? We cannot use `$client1->getSendingRequestCount()` or ` $client1->isDisabled()` because those methods are final